### ISSUE TITLE
Array builder not working in IPython console of spyder 3

### ISSUE THREAD

**Issue by gautham949** [spyder-ide/spyder#5196]
I try to use the shortcut key to Ctrl + M to enter the array builder in the console but it always shows the following error. What does it mean and how to fix it? Can anyone help me understand the problem

  File "/home/gautham/anaconda3/lib/python3.6/site-packages/spyder/widgets/ipythonconsole/shell.py", line 263, in <lambda>
    array_table = config_shortcut(lambda: self.enter_array_table(),
AttributeError: 'ShellWidget' object has no attribute 'enter_array_table'

## Version and main components

* Spyder Version: 3.2.1 
* Python Version: 3.6.1
* Qt Versions: 5.6.2, PyQt5 5.6 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.1 (OK)
numpy >=1.7      :  1.12.1 (OK)
sphinx >=0.6.6   :  1.5.6 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.2.2 (OK)
nbconvert >=4.0  :  5.1.1 (OK)
sympy >=0.7.3    :  1.0 (OK)
cython >=0.21    :  0.25.2 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
IPython >=4.0    :  5.3.0 (OK)
pylint >=0.25    :  1.6.4 (OK)

```

**Comment by ccordoba12** [comment]
This was fixed in Spyder 3.2.2. Please update to our latest version to get the fix.

**Comment by gautham949** [comment]
Thank you. Updated it to 3.2.2 and it is fixed.
